src/appleseed/foundation/image/icanvas.h

```diff
@@ -175,15 +176,15 @@ inline const uint8* ICanvas::pixel(
 #define FOUNDATION_CHECK_PIXEL_SIZE(color) \
```

Comment:
Will this macro still be required if you modify `get_pixel()`? If not, please remove it.